gqlgenc.yml

```diff
@@ -0,0 +1,29 @@
+model:
+  package: graphclient
+  filename: ./models_gen.go
+client:
+  package: graphclient
+  filename: ./client.go # Where should any generated client go?
+models:
+  ID:
+    model:
+      - go.infratographer.com/x/gidx.PrefixedID
+  Int:
+    model: github.com/99designs/gqlgen/graphql.Int64
+  Date:
+    model: github.com/99designs/gqlgen/graphql.Time
+  JSON:
+    model:
+      - go.infratographer.com/x/entx.RawMessage
+# schema:
+#   - "schema/*.graphql"
+# The client is supposed to support building from files using the config above ^
+# Because of the federation support in our schema this fails and doesn't work.
+# Instead pointing at the local instance of our application as that does work.
+endpoint:
+  url: http://localhost:7905/query
```

Comment:
Since a running server is required, we should make it run in the makefile. Capturing the pid with bash is fine.